src/views/virtualmachines/details/components/snapshots/SnapshotList.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import {
+  V1alpha1VirtualMachineRestore,
+  V1alpha1VirtualMachineSnapshot,
+  V1VirtualMachine,
+} from '@kubevirt-ui/kubevirt-api/kubevirt';
+import {
+  VirtualMachineRestoreModelGroupVersionKind,
+  VirtualMachineSnapshotModelGroupVersionKind,
+} from '@kubevirt-utils/models';
```

Comment:
you can get it from the API repo
https://kubevirt-ui.github.io/kubevirt-api/modules/console.html#VirtualMachineSnapshotModelGroupVersionKind

Reply:
changed, thanks :)